src/pages/K8s.vue

```diff
@@ -446,12 +453,19 @@ export default {
   display: grid;
   gap: 1em;
   grid-template-areas:
-    "version  port"
-    "kubernetes features";
+    "version port"
+    "kubernetes etc"
+    "traefik flannel";
```

Comment:
We're working on redesigning this page, so I think an acceptable temporary measure might be to expand the grid into 6 columns/2 rows so that we can evenly distribute our version and port inputs in first row with something like this

```suggestion
    "version version version port port port"
    "kubernetes kubernetes traefik traefik flannel flannel";
```